python/ray/air/_internal/tensorflow_utils.py

```diff
@@ -152,3 +59,41 @@ def convert_ndarray_batch_to_tf_tensor_batch(
         }
 
     return batch
+
+
+def get_type_spec(
+    schema: "pyarrow.lib.Schema",
```

Comment:
TODO: This should be `pyarrow.lib.Schema | PandasSchema` or whatever

Reply:
Yep that should be a simple type annotation update, looks like the underlying implementation already works for both types!